### PR TITLE
Add feature specs for more discovery report end states

### PR DESCRIPTION
**Pull request by ndushay**
## Why was this change made? 🤔

Better feature tests decrease the human testing burden.

Part of #942

## How was this change tested? 🤨

⚡ ⚠ If this change has cross service impact (e.g. changes what is written to shared file systems or how it uses APIs), ***run [integration tests](https://github.com/sul-dlss/infrastructure-integration-test)*** and/or test in [stage|qa] environment, in addition to specs. ⚡

### CODE REVIEW

File: spec/features/discovery_report/build_manifest_image_spec.rb [hunk 1]
```diff
@@ -6,17 +6,13 @@
   let(:project_name) { "discovery-report-image-#{RandomWord.nouns.next}" }
   let(:bundle_dir) { Rails.root.join('spec/test_data/image_jpg') }
   let(:bare_druid) { 'pr666rr9999' }
-  let(:object_staging_dir) { Rails.root.join(Settings.assembly_staging_dir, 'pr', '666', 'rr', '9999', bare_druid) }
```
Comment (ndushay) [inline spec/features/discovery_report/build_manifest_image_spec.rb:9]: staging directory not used for discovery reports.

**Review by ndushay — COMMENTED** [review]
@peetucket I note that my new test_data overlaps with yours; in your PR, can you make the new failure test use your folder and get rid of the new one here?

**Review by peetucket — APPROVED** [review]
it looks like you reconciled the test manifest data from the recently merged PR, so looks good to me if so

**Comment by ndushay** [comment]
yep.  merging

**Pull request merged**